v1/examples/pipeline/steps/service-now-approval-step.yaml

```diff
@@ -0,0 +1,26 @@
+version: 1
+kind: pipeline
+spec:
+  stages:
+    - type: approval
+      name: approval
+      id: Approval
+      timeout: 10m
+      when: <+<+environment.name> != "QA">
+      spec: 
+        steps:
+          - type: service-now-approval
+            name: serviceNow
+            id: serviceNow
+            spec: 
+              connector: SNOWConnector
+              number: "123"
```

Comment:
should we call it ticket instead of number?
or we could have it like 
```
ticket:
  number: "123"
  type: change
```

Reply:
Done